src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/HiddenFieldCheckTest.java

```diff
@@ -470,6 +470,26 @@ public void testClassNestedInRecord() throws Exception {
             expected);
     }
 
+    @Test
+    public void testHiddenFieldRecords2() throws Exception {
```

Comment:
```suggestion
    public void testHiddenFieldRecordsImplicitlyStatic() throws Exception {
```

Reply:
@Kevin222004 meaningfully named tests/test inputs are a clear signal that each test serves a specific purpose and helps to define expected behavior before we ever even see test input or config.

Reply:
done